### DISCUSSION TITLE
Configuring renovate in a multi-project Gradle build

### DISCUSSION THREAD

**Discussion by roded** [renovatebot/renovate#20593] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

N/A

### Wanted end result.

Hi,
I have a mono-repro with a multi-project Gradle build.
For every project inside the repository, I'd like to be able to control renovatebot's configuration (which directories to scan, what managers to enable, how the commit messages look like, PR count, etc.). I'd like renovate to create a PR per dependency in each project (and not combine dependency version bumps from multiple projects even for the same dependencies).
I'm somewhat at loss regarding to to accomplish this, any help or examples would be much appreciated.

Hope I didn't miss a similar Q&A here.
Thank you. 

### What you tried so far.

So far, my single renovate.json file in the repository root looks like:

```
{
    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
    "extends": [
        "config:base"
    ],
    "prConcurrentLimit": 3,
    "prHourlyLimit": 3,
    "hostRules": [
        {
            "enabled": false,
            "matchHost": "http://nexus.company.lab:8081"
        }
    ],
    "baseBranches": [
        "develop"
    ],
    "enabledManagers": [
        "gradle",
        "gradle-wrapper",
        "dockerfile"
    ],
    "includePaths": [
        "gradle/projectalibs.versions.toml",
        "project-a/**,",
        "gradle/projectblibs.versions.toml",
        "project-b/**,"
    ],
    "commitBodyTable": true,
    "packageRules": [
        {
            "paths": [
                "gradle/projectalibs.versions.toml",
                "project-a/**,",
            ],
            "assignees": [
                "roded"
            ],
            "commitBody": "[projecta-test]",
            "commitMessagePrefix": "PROJA-0000: ",
            "prConcurrentLimit": 3
        },
        {
            "paths": [
                "gradle/projectblibs.versions.toml",
                "project-b/**,",
            ],
            "assignees": [
                "roded"
            ],
            "commitBody": "[projectb-test]",
            "commitMessagePrefix": "PROJB-0000: ",
            "prConcurrentLimit": 1
        }
    ]
}
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
use additional branch prefix to create separate PR per folder.

PR limits are only configurable for repo.

**Reply by roded** [reply]
Thanks @viceice, I'll give it a shot.
